script.js

```diff
@@ -0,0 +1,103 @@
+let myLibrary = [];
+let table = document.querySelector("table");
+const newBtn = document.querySelector("#new-btn");
+const form = document.querySelector("form");
+
+function Book(title, author, pages, readStatus) {
+    this.title = title,
+    this.author = author,
+    this.pages = pages,
+    this.readStatus = readStatus;
+}
+
+Book.prototype.changeReadStatus = function() {
+    if (this.readStatus === "Read") {
+        this.readStatus = "Unread";
+    }
+    else{
+        this.readStatus = "Read";
+    }
+}
+
+const theHobbits = new Book(
+    "The Hobbits",
+    "J.R.R. Tolkiens",
+    295, 
+    "Read"
+);
+
+const harryPotter = new Book(
+  "Harry Potter and The Cursed Child",
+  "J.K. Rowling",
+  345,
+  "Unread"
+);
+
+myLibrary.push(theHobbits, harryPotter);
+
+const addBookToLibrary = () => {
+  let title = document.querySelector("#book-name").value;
+  let author = document.querySelector("#author-name").value;
+  let pages = document.querySelector("#pages").value;
+  let readStatus = document.querySelector(
+    'input[name="read_status"]:checked'
+  ).value;
+  let newBook = new Book(title, author, pages, readStatus);
+  myLibrary.push(newBook);
+  displayBooks();
+  form.style.display = "none";
+}
+
+const displayBooks = () => {
+  if (myLibrary.length > 0) {
+      table.innerHTML = `
+      <tr class='table-heading'>
+      <th>Book Name</th>
+      <th>Author Name</th>
+      <th>Pages</th>
```

Comment:
user indentation, for the sake of readability :)

Reply:
fixed